scripts/dev/stack-info.sh

```
@@ -0,0 +1,99 @@
+    printf "\nResolved local endpoints:\n"
+    print_url "N1" "${n1_port:+http://localhost:${n1_port}}"
+    print_url "Medusa admin/API" "${medusa_port:+http://localhost:${medusa_port}/app}"
+    print_url "Meilisearch" "${meili_port:+http://localhost:${meili_port}/health}"
+    print_url "MinIO console" "${minio_console_port:+http://localhost:${minio_console_port}}"
+    print_url "MinIO API" "${minio_api_port:+http://localhost:${minio_api_port}}"
+    print_url "Postgres" "${postgres_port:+localhost:${postgres_port}}"
+    print_url "Adminer" "${adminer_port:+http://localhost:${adminer_port}}"
+    print_url "Caddy HTTP" "${caddy_http_port:+$(local_http_url n1.medusa.localhost "$caddy_http_port")}"
+    print_url "Caddy HTTPS" "${caddy_https_port:+$(local_https_url n1.medusa.localhost "$caddy_https_port")}"
+  else
+    printf "\nNo runtime env exists yet. Run `mise run dev:ports` or `mise run dev`.\n"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)